templates/keystoneapi/config/keystone-api-config.json

```diff
@@ -81,7 +81,16 @@
             "owner": "keystone:apache",
             "perm": "0444",
             "optional": true
+        },
+        {
+            "source": "/var/lib/config-data/default/multirealm-federation/*",
+            "dest": "/var/lib/httpd/metadata/",
+            "owner": "keystone:apache",
+            "perm": "0644",
```

Comment:
should this be 640?

The whole point behind this change was to have the files owned by keystone:apache - and readable/usable only by those users.  Given that this file contains some passwords (client_password), it probably shouldn't be readable by other.

Incidentally, I do think that its likely that without this patch, the files would have been readable because they were owned by root but had global read permissions.  I  think the problem was that we were trying to test with the CLI - and that just doesn't work.